.changeset/moody-apricots-kiss.md

```diff
@@ -0,0 +1,5 @@
+---
+"@sveltejs/kit": patch
+---
+
+fix: respect `enctype`/`formenctype` HTML form attributes from form + set `application/x-www-form-urlencoded` as a default `Content-Type` for `POST` requests made by `enhance`
```

Comment:
```suggestion
fix: respect HTML attributes `enctype` and `formenctype` for forms with `use:enhance`
```

Might be better to generate a second patch changeset since there are two changes:
```
fix: set default `Content-Type` header to `application/x-www-form-urlencoded` for `POST` form submissions with `use:enhance` to align with native form behaviour
```

Reply:
Addressed with 63af8eec634006b2bfef1aee63ce6af4bbd7dddc